requirements/dev.in

```diff
@@ -4,6 +4,6 @@
 
 -r pip-tools.txt          # pip-tools and its dependencies, for managing requirements files
 -r quality.txt            # Core and quality check dependencies
--r travis.txt             # tox and related dependencies
```

Comment:
This line was causing the make upgrade to fail as this file had been renamed to `ci.txt`